geopyspark/rdd.py

```diff
@@ -57,10 +59,20 @@ def __init__(self, geopysc, rdd_type, srdd):
         self.rdd_type = rdd_type
         self.srdd = srdd
 
+    @property
```

Comment:
What is the purpose of the `@property` here? I'm not sure if there is extra magic that is happening?